Formula/crystal-lang.rb

```diff
@@ -6,25 +6,8 @@ class CrystalLang < Formula
   head "https://github.com/crystal-lang/crystal.git"
 
   stable do
```

Comment:
Ah, thank you. I had no idea what that did because I didn't put it there in the first place ( :-( ). I'll make the change right now and rebase this PR.